demo/api/src/products/entities/product.entity.ts

```diff
@@ -198,4 +198,12 @@ export class Product extends BaseEntity<Product, "id"> {
 
     @ManyToOne(() => Manufacturer, { nullable: true, index: true, ref: true })
     manufacturer?: Ref<Manufacturer> = undefined;
+
+    @ManyToOne(() => PublicUpload, { nullable: true, ref: true })
+    @Field(() => PublicUpload, { nullable: true })
+    factsheet?: Ref<PublicUpload> = undefined;
+
+    @ManyToMany(() => PublicUpload)
+    @Field(() => [PublicUpload])
+    datasheets = new Collection<PublicUpload>(this);
```

Comment:
Isn't factsheet and datasheet basically the same? 😁

Reply:
More or less 😅 
I just wanted to add examples for single and multi file uploads. 

I'm not sure if this is the best approach though, as the products example will become quite bloated when adding every feature comet supports to this example 🤔

Reply:
Should we add another structured content example for this?

Reply:
Maybe for now we can change this to e.g. a `pricelist` single file upload and a `datasheets` multi file upload. 
Then it somewhat makes sense and we can always add new structured content in the future, if we want to showcase other features individually as well.

Reply:
Sounds good!

Reply:
Done